x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecation_logs/es_deprecation_logs.test.tsx

```diff
@@ -41,18 +41,16 @@ const getLoggingResponse = (toggle: boolean): DeprecationLoggingStatus => ({
 
 describe('ES deprecation logs', () => {
   let testBed: EsDeprecationLogsTestBed;
-  const { server, httpRequestsMockHelpers } = setupEnvironment();
-
+  let mockEnvironment: ReturnType<typeof setupEnvironment>;
   beforeEach(async () => {
-    httpRequestsMockHelpers.setLoadDeprecationLoggingResponse(getLoggingResponse(true));
-    testBed = await setupESDeprecationLogsPage();
+    mockEnvironment = setupEnvironment();
+    mockEnvironment.httpRequestsMockHelpers.setLoadDeprecationLoggingResponse(
```

Comment:
What do you think of destructing `httpRequestsMockHelpers` from `mockEnvironment`? I think that will reduce the noise of the diff and also make things easier to follow since the changes will be scoped to the setup of the test rather the implementation of each.

I think this change can be applied throughout most of the test files in this PR.

Reply:
Yep, good point, let me do that!